test/switch-test.js

```diff
@@ -15,6 +15,19 @@ describe('<Switch/>', function() {
       expect(component.find('.bootstrap-switch-handle-on').length).to.equal(1);
       expect(component.find('.bootstrap-switch-handle-off').length).to.equal(0);
     });
+    it('onLabel is ON by default', function() {
+      expect(component.find('div.switch-container').text()).to.contain('ON');
+    });
+  });
```

Comment:
This PR looks good! Can we add one more test to test that offLabel is OFF by default? After that this is good to merge!